Procfile

```diff
@@ -0,0 +1,4 @@
+hydra-server: ./scripts/start-hydra.sh
+hydra-queue-runner: ./scripts/start-queue-runner.sh
+hydra-evaluator: ./scripts/start-evaluator.sh
+postgres: ./scripts/start-postgres.sh
```

Comment:
Could these be moved to another directory (e.g. `./foreman`)? Currently the naming suggests that these are *the* scripts for running Hydra. And it creates some ambiguity about where to place scripts given that `src/script` already exists.

Reply:
Okay no problem.

Reply:
done.